Gemfile

```diff
@@ -25,6 +25,10 @@ gem 'bootsnap', '>= 1.4.4', require: false
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
 # gem 'rack-cors'
 
+# gem 'dotenv', '~> 2.7.6'
+gem 'dotenv-rails', groups: [:development, :test]
```

Comment:
When I need to lock a gem? Always?

Reply:
Yes, it is needed to avoid having different versions in different environments.
My suggestion is when you are adding a new gem put it in the gemfile, see the version installed and after that lock it in the gemfile.